webcompat/static/js/lib/issues.js

```diff
@@ -484,6 +485,12 @@ issues.MainView = Backbone.View.extend({
   },
   render: function() {
     this.$el.fadeIn();
+  },
+
+  keyAction: function(e) {
+    if (e.keyCode === 13 && e.ctrlKey ) { 
```

Comment:
Since we're already shipping [Mousetrap](https://github.com/webcompat/webcompat.com/blob/2ca6d388907ec1f721e651df518857ff851d99e6/webcompat/templates/layout.html#L57), maybe we can use that: https://craig.is/killing/mice